components/monitoring/grafana/base/dashboards/image-controller/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+#TODO
```

Comment:
Is this waiting for another PR?

Reply:
yes for https://github.com/redhat-appstudio/image-controller/pull/85

Reply:
updated